<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="0" total="6" passed="3" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-10-16T21:18:16 IST" name="Suite" finished-at="2025-10-16T21:18:56 IST" duration-ms="39020">
    <groups>
      <group name="submitorder">
        <method signature="StandaloneTest.submitOrderTest(java.util.HashMap)[pri:0, instance:SelFrameWork.StandaloneTest@28cda624]" name="submitOrderTest" class="SelFrameWork.StandaloneTest"/>
      </group> <!-- submitorder -->
      <group name="smoke">
        <method signature="ErrorValidationTest.incorrectEmailOrPassword()[pri:0, instance:SelFrameWork.ErrorValidationTest@6be968ce]" name="incorrectEmailOrPassword" class="SelFrameWork.ErrorValidationTest"/>
      </group> <!-- smoke -->
    </groups>
    <test started-at="2025-10-16T21:18:16 IST" name="ErrorValidationTest" finished-at="2025-10-16T21:18:46 IST" duration-ms="29047">
      <class name="SelFrameWork.ErrorValidationTest">
        <test-method is-config="true" signature="launchOpenPage()[pri:0, instance:SelFrameWork.ErrorValidationTest@6be968ce]" started-at="2025-10-16T21:18:17 IST" name="launchOpenPage" finished-at="2025-10-16T21:18:27 IST" duration-ms="10010" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchOpenPage -->
        <test-method signature="LoginErrorTest()[pri:0, instance:SelFrameWork.ErrorValidationTest@6be968ce]" started-at="2025-10-16T21:18:27 IST" name="LoginErrorTest" finished-at="2025-10-16T21:18:27 IST" duration-ms="432" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:SelFrameWork.ErrorValidationTest@6be968ce]" started-at="2025-10-16T21:18:27 IST" name="tearDown" finished-at="2025-10-16T21:18:28 IST" duration-ms="1348" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchOpenPage()[pri:0, instance:SelFrameWork.ErrorValidationTest@6be968ce]" started-at="2025-10-16T21:18:28 IST" name="launchOpenPage" finished-at="2025-10-16T21:18:35 IST" duration-ms="6961" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchOpenPage -->
        <test-method retried="true" signature="incorrectEmailOrPassword()[pri:0, instance:SelFrameWork.ErrorValidationTest@6be968ce]" started-at="2025-10-16T21:18:35 IST" name="incorrectEmailOrPassword" finished-at="2025-10-16T21:18:36 IST" duration-ms="355" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email o password.] but found [Incorrect email or password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email o password.] but found [Incorrect email or password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at SelFrameWork.ErrorValidationTest.incorrectEmailOrPassword(ErrorValidationTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- incorrectEmailOrPassword -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:SelFrameWork.ErrorValidationTest@6be968ce]" started-at="2025-10-16T21:18:36 IST" name="tearDown" finished-at="2025-10-16T21:18:37 IST" duration-ms="1134" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchOpenPage()[pri:0, instance:SelFrameWork.ErrorValidationTest@6be968ce]" started-at="2025-10-16T21:18:37 IST" name="launchOpenPage" finished-at="2025-10-16T21:18:44 IST" duration-ms="7107" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchOpenPage -->
        <test-method signature="incorrectEmailOrPassword()[pri:0, instance:SelFrameWork.ErrorValidationTest@6be968ce]" started-at="2025-10-16T21:18:44 IST" name="incorrectEmailOrPassword" finished-at="2025-10-16T21:18:44 IST" duration-ms="250" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email o password.] but found [Incorrect email or password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email o password.] but found [Incorrect email or password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at SelFrameWork.ErrorValidationTest.incorrectEmailOrPassword(ErrorValidationTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- incorrectEmailOrPassword -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:SelFrameWork.ErrorValidationTest@6be968ce]" started-at="2025-10-16T21:18:44 IST" name="tearDown" finished-at="2025-10-16T21:18:46 IST" duration-ms="1062" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- SelFrameWork.ErrorValidationTest -->
    </test> <!-- ErrorValidationTest -->
    <test started-at="2025-10-16T21:18:16 IST" name="StandaloneTest" finished-at="2025-10-16T21:18:56 IST" duration-ms="39020">
      <class name="SelFrameWork.StandaloneTest">
        <test-method is-config="true" signature="launchOpenPage()[pri:0, instance:SelFrameWork.StandaloneTest@28cda624]" started-at="2025-10-16T21:18:17 IST" name="launchOpenPage" finished-at="2025-10-16T21:18:26 IST" duration-ms="9160" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchOpenPage -->
        <test-method signature="submitOrderTest(java.util.HashMap)[pri:0, instance:SelFrameWork.StandaloneTest@28cda624]" started-at="2025-10-16T21:18:26 IST" name="submitOrderTest" data-provider="getData" finished-at="2025-10-16T21:18:31 IST" duration-ms="5270" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Hari9490@, product=ZARA COAT 3, email=surampudi@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrderTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:SelFrameWork.StandaloneTest@28cda624]" started-at="2025-10-16T21:18:31 IST" name="tearDown" finished-at="2025-10-16T21:18:33 IST" duration-ms="1364" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchOpenPage()[pri:0, instance:SelFrameWork.StandaloneTest@28cda624]" started-at="2025-10-16T21:18:33 IST" name="launchOpenPage" finished-at="2025-10-16T21:18:40 IST" duration-ms="7346" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchOpenPage -->
        <test-method signature="submitOrderTest(java.util.HashMap)[pri:0, instance:SelFrameWork.StandaloneTest@28cda624]" started-at="2025-10-16T21:18:40 IST" name="submitOrderTest" data-provider="getData" finished-at="2025-10-16T21:18:45 IST" duration-ms="5089" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Pavani1008@, product=ADIDAS ORIGINAL, email=pavani1008@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrderTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:SelFrameWork.StandaloneTest@28cda624]" started-at="2025-10-16T21:18:45 IST" name="tearDown" finished-at="2025-10-16T21:18:47 IST" duration-ms="1644" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchOpenPage()[pri:0, instance:SelFrameWork.StandaloneTest@28cda624]" started-at="2025-10-16T21:18:47 IST" name="launchOpenPage" finished-at="2025-10-16T21:18:54 IST" duration-ms="6979" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchOpenPage -->
        <test-method signature="productConfirmation()[pri:0, instance:SelFrameWork.StandaloneTest@28cda624]" started-at="2025-10-16T21:18:54 IST" name="productConfirmation" finished-at="2025-10-16T21:18:54 IST" depends-on-methods="SelFrameWork.StandaloneTest.submitOrderTest" duration-ms="465" status="FAIL">
          <exception class="org.openqa.selenium.ElementClickInterceptedException">
            <message>
              <![CDATA[Element <button class="btn btn-custom"> is not clickable at point (1175,27) because another element <div class="ngx-spinner-overlay ng-tns-c31-1 ng-trigger ng-trigger-fadeIn ng-star-inserted ng-animating"> obscures it
Build info: version: '4.35.0', revision: '1c58e5028b'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.12'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Command: [a411ee6a-3478-4ea9-b405-26813725922f, clickElement {id=5a806000-2398-4cf5-ae78-7bcec3f3e8ad}]
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 144.0, moz:accessibilityChecks: false, moz:buildID: 20251009125714, moz:geckodriverVersion: 0.36.0, moz:headless: false, moz:platformVersion: 10.0, moz:processID: 10840, moz:profile: C:\Users\JESUS\AppData\Loca..., moz:shutdownTimeout: 60000, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, userAgent: Mozilla/5.0 (Windows NT 10....}
Element: [[FirefoxDriver: firefox on windows (a411ee6a-3478-4ea9-b405-26813725922f)] -> xpath: //ul//li//button[@routerlink='/dashboard/myorders']]
Session ID: a411ee6a-3478-4ea9-b405-26813725922f]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.ElementClickInterceptedException: Element <button class="btn btn-custom"> is not clickable at point (1175,27) because another element <div class="ngx-spinner-overlay ng-tns-c31-1 ng-trigger ng-trigger-fadeIn ng-star-inserted ng-animating"> obscures it
Build info: version: '4.35.0', revision: '1c58e5028b'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.12'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Command: [a411ee6a-3478-4ea9-b405-26813725922f, clickElement {id=5a806000-2398-4cf5-ae78-7bcec3f3e8ad}]
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 144.0, moz:accessibilityChecks: false, moz:buildID: 20251009125714, moz:geckodriverVersion: 0.36.0, moz:headless: false, moz:platformVersion: 10.0, moz:processID: 10840, moz:profile: C:\Users\JESUS\AppData\Loca..., moz:shutdownTimeout: 60000, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, userAgent: Mozilla/5.0 (Windows NT 10....}
Element: [[FirefoxDriver: firefox on windows (a411ee6a-3478-4ea9-b405-26813725922f)] -> xpath: //ul//li//button[@routerlink='/dashboard/myorders']]
Session ID: a411ee6a-3478-4ea9-b405-26813725922f
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:215)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:557)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:223)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:76)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at jdk.proxy2/jdk.proxy2.$Proxy13.click(Unknown Source)
at sel.abstractComponents.AbstractComponents.orderPageClick(AbstractComponents.java:54)
at SelFrameWork.StandaloneTest.productConfirmation(StandaloneTest.java:50)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.ElementClickInterceptedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- productConfirmation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:SelFrameWork.StandaloneTest@28cda624]" started-at="2025-10-16T21:18:54 IST" name="tearDown" finished-at="2025-10-16T21:18:56 IST" duration-ms="1164" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- SelFrameWork.StandaloneTest -->
    </test> <!-- StandaloneTest -->
  </suite> <!-- Suite -->
</testng-results>
